src/NServiceBus.Core/Unicast/Messages/DeserializeLogicalMessagesBehavior.cs

```diff
@@ -97,12 +92,6 @@ List<LogicalMessage> Extract(TransportMessage physicalMessage)
             }
         }
 
-        [ObsoleteEx(RemoveInVersion = "6.0")]
-        bool DoesTypeHaveImplAddedByVersion3(string existingTypeString)
```

Comment:
So basically I need to pull that in again. I'll do right after the talk with danny.